packages/chart/src/Chart.module.scss

```diff
@@ -1,4 +1,4 @@
-@import '@deephaven/components/scss/custom.scss';
+@import '../../components/scss/custom.scss';
```

Comment:
I don't really like having to do a relative dir for components like this - it makes it more fragile, if we move one of these scss files or one of those folders we break the chain.

Did you take a look at the sass watch source at all to see if there's any way to restrict the types of files it actually watches? Would love to have an upstream fix (or issue) filed as well.

Reply:
I found where they have it hardcoded to ignore files that aren't sass... https://github.com/sass/dart-sass/blob/f6819060ec54358e318fafa6395238975030834c/lib/src/executable/watch.dart#L129

```
              if (extension != '.sass' &amp;&amp; extension != '.scss' &amp;&amp; extension != '.css') {
                continue;
              }
```

Reply:
I think I'd be using chokidar, so this watch? https://github.com/sass/dart-sass/blob/f6819060ec54358e318fafa6395238975030834c/lib/src/io/node.dart#L218

I wonder if it _would_ be better with globbing (specifying **/*.scss or something). They mention being judicious about dirs you're watching as it may recursively watch a lot: https://github.com/paulmillr/chokidar#how

They've also got another watch impl here but I don't think I'm hitting this one: https://github.com/sass/dart-sass/blob/f6819060ec54358e318fafa6395238975030834c/lib/src/io/vm.dart#L90

There's also a couple relevant issues that we could track/add to:
https://github.com/sass/dart-sass/issues/1292
https://github.com/sass/sass/issues/2739 (which speaks about a fix not requiring modifying load-paths)

Reply:
These could be kept for now. There's 3 or 4 packages which need to watch their local node_modules and add a few thousand watchers each.

Could also specifically watch `node_modules/@deephaven/components/scss` but then the imports are less clear since it would just be `import 'custom.scss'`

Pretty sure I tried watching globs and it didn't work. It's also really annoying because from the source it looks like it should detect only sass files first and then only pass those to watch, but that isn't working on Linux it seems.